### DISCUSSION TITLE
When multiple series on a chart, add to alerting template the name of the series that triggered the alert

### DISCUSSION THREAD

**Discussion by waterdrop01** [grafana/grafana#37090] [Alerting]
Hello,

I have this kind of charts in Grafana v8.0.5:
![node load](https://user-images.githubusercontent.com/47316824/126611130-f0710fe2-8582-483e-96b5-79afb3e52df3.png)

And I created this alert:
![Screenshot from 2021-07-22 10-06-49](https://user-images.githubusercontent.com/47316824/126611154-b0a90bfa-1edf-415a-8457-c1195ff49767.png)

When one of the series cross over 90%, Grafana new alerting system is sending an alert.

I would like this alert to include the name of the specific series responsible for triggering the alert (the one the crossed over 90%)

I don't think we can do that currently, and this would make the alerts content so more useful.

Thanks!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
